content/blog/2024/01/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Camel 2023 in Numbers"
+date: 2023-01-03
+authors: [davsclaus]
+categories: ["Roadmap"]
+preview: Quick summary of the Apache Camel project in 2023 by numbers
+---
+
+It's the time of the year when we take a look back at 2023, and compile a brief summary (by numbers) of the Apache Camel project(s).
+
+## Camel 2023 in Numbers
+
+TODO: Update with new numbers
+
+Number of Camel Core releases in 2022: **19**
+
+Number of Camel Quarkus releases in 2022: **15**
+
+Number of Camel K releases in 2022: **11**
+
+Number of Camel Kafka Connector releases in 2022: **4**
+
+Number of commits in 2022: **6162** [1]
+
+Total number of JIRA tickets created at end of 2022: **18825**
+
+Number of JIRA tickets created in 2022: **1400**
+
+Number of JIRA tickets resolved in 2022: **1700**
+
+Number of stars on GitHub at end of 2022: **4652**
+
+Total number of commits at end of 2022: **61875**
+
+Total number of contributors on GitHub at end of 2022: **923**
+
+Number of individual committers doing commits in 2022: **212** [2]
+
+Number of closed pull requests at end of 2022: **8933**
+
+Number of closed pull requests in 2022: **2344**
+
+Number of Twitter followers at end of 2022: **2759**
```

Comment:
Need to rename to `X`.

And I feel like a fair part fo the activity has been moved to LinkedIn Apache Camel group. Maybe worth adding it